Sofa/Component/IO/Mesh/src/sofa/component/io/mesh/MeshSTLLoader.h

```diff
@@ -48,8 +48,16 @@ class SOFA_COMPONENT_IO_MESH_API MeshSTLLoader : public sofa::core::loader::Mesh
 
 public:
     //Add Data here
-    Data <unsigned int> _headerSize; ///< Size of the header binary file (just before the number of facet).
-    Data <bool> _forceBinary; ///< Force reading in binary mode. Even in first keyword of the file is solid.
+    SOFA_ATTRIBUTE_DEPRECATED__RENAME_DATA()
+    Data <bool> forceBinary;
```

Comment:
```suggestion
    Data <bool> _forceBinary;
```
It previously started with an underscore